File: two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        //edge case
        if(nums == null || nums.length == 0){
            return new int[]{-1,-1};
        }
        //logic
        for(int i = 0; i < nums.length; i++){
            for(int j = i+1; j < nums.length; j++){
                if(nums[i] + nums[j] == target){
                    return new int[]{i,j};
                }
            }
        }
        
        return new int[]{-1,-1};
    }
}

File: two-sum/two-sum.py
class Solution:
    def twoSum(self, nums, target):
        #create an empty hashmap
        hashmap = {}
        for k,v in enumerate(nums): #run through the entire array
            b = target - nums[k] #check for the difference between target and every element in k
            if b in hashmap:#for every value in hashmap
                return [hashmap[b],k] #return the hashmap[key,val]
            hashmap[v] = k  #else populate the value in hashmap
        return
